src/resources/js/components/Presentation.vue

```diff
@@ -1,50 +1,70 @@
 <template>
```

Comment:
If for some technical reason you can't follow the models with a link to create a new poll, you need to move the block to create a poll above all the polls.